Frameworks/UIKit/UIButton.mm

```diff
@@ -167,19 +169,29 @@ - (void)_initUIButton {
         FAIL_FAST();
     }
 
+    // Default to a custom button type
+    _buttonType = UIButtonTypeCustom;
+
+    // Default to showing pressed/disabled states
+    self.adjustsImageWhenDisabled = YES;
+    self.adjustsImageWhenHighlighted = YES;
+
     // Force-load the template, and get the TextBlock and Image for use in our proxies.
     [_xamlButton applyTemplate];
-
+    [_xamlButton updateLayout];
+
+    // Create our child UILabel; its frame will be updated in layoutSubviews
+    // TODO: Ideally we'd grab this directly from the Xaml, but we're not able to launch some apps when doing so due to a
```

Comment:
yeah, bummer. it actually worked fine with WoCCatalog and XamlCatalog; must have something to do with our import flow.  

---
In reply to: [100646740](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r100646740) [](ancestors = 100646740)